velox/functions/prestosql/aggregates/AverageAggregate.cpp

```diff
@@ -398,18 +450,20 @@ exec::AggregateRegistrationResult registerAverage(const std::string& name) {
                            .argumentType("real")
                            .build());
 
-  signatures.push_back(exec::AggregateFunctionSignatureBuilder()
-                           .integerVariable("a_precision")
-                           .integerVariable("a_scale")
-                           .argumentType("DECIMAL(a_precision, a_scale)")
-                           .intermediateType("varbinary")
-                           .returnType("DECIMAL(a_precision, a_scale)")
-                           .build());
+  if (aggregateType == AggregateType::kAvg) {
```

Comment:
Please add a comment here indicating that Decimal types are only supported for Avg and not Geometric mean.